articles/hdinsight/hbase/apache-hbase-tutorial-get-started-linux.md

```diff
@@ -24,7 +24,7 @@ Before you begin trying this HBase example, you must have the following items:
 * [Secure Shell(SSH)](../hdinsight-hadoop-linux-use-ssh-unix.md). 
 * [curl](http://curl.haxx.se/download.html).
 
-## Create HBase cluster
+## Create Apache HBase cluster

```

Comment:
We are creating an HDInsight cluster of type HBase. I think the Apache part might be confusing. 
![image](https://user-images.githubusercontent.com/5067358/49033640-92664680-f164-11e8-833e-074eac58c92e.png)